### PR TITLE
docs: Create SECURITY.md

### PR DESCRIPTION
**Pull request by DawoudSheraz**
- Created an initial security policy for Tutor.

### CODE REVIEW

**Review by qasim-arbisoft — COMMENTED** [review]
- Name it as Tutor Ethical Vulnerability Disclosure Policy
- Also, it should let them know whether Tutor/Edly runs any bug bounty program or not

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Tutor Security Policy
```
Comment (ziafazal) [inline SECURITY.md:1]: Can we rename it to `Name it as Tutor Ethical Vulnerability Disclosure Policy`

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+* The nature of the vulnerability, e.g.
+  * Authentication and Authorization
+  * Data Integrity and Confidentiality
+  * Security Configurations
+  * Third-party dependencies
+* The impact of the security risk
+* A detailed description of the steps necessary to reproduce the issue
+* The links to the vulnerable code
+* The links to third-party libraries/packages if the vulnerability is present in such a dependency.
```
Comment (ziafazal) [inline SECURITY.md:16]: Can we add a para graph at the end about whether Tutor/Edly runs any bug bounty program or not?

**Review by ziafazal — COMMENTED** [review]
@DawoudSheraz it appears Qasim has reviewed it already.

**Comment by DawoudSheraz** [comment]
> * Name it as Tutor Ethical Vulnerability Disclosure Policy
> * Also, it should let them know whether Tutor/Edly runs any bug bounty program or not

Sure, thank you.

**Review by regisb — APPROVED** [review]

**Review by qasim-arbisoft — APPROVED** [review]
Looks good to me. This needs to be reviewed annually.

**Pull request merged**